google/datalab/contrib/mlworkbench/_local_predict.py

```diff
@@ -88,9 +94,14 @@ def _download_images(data, img_cols):
   for d in data:
     for img_col in img_cols:
       if d.get(img_col, None):
-        with file_io.FileIO(d[img_col], 'rb') as fi:
-          im = Image.open(fi)
-        images[img_col].append(im)
+        if isinstance(d[img_col], Image.Image):
```

Comment:
I added something in get_prediction_results's docstring. Also, I think we will update our notebook sample with images to show how to build a prediction client.